sink-connector/python/db_load/clickhouse_loader.py

```diff
@@ -202,8 +204,8 @@ def convert_to_clickhouse_table(user_name, table_name, source):
     src = re.sub(r'', '', src)
     # adding virtual columns ver and sign
     src = re.sub(r'\) ENGINE',
-                 '  `_sign` Int8 DEFAULT 1,\n  `_version` UInt64 DEFAULT 0\n) ENGINE', src)
```

Comment:
we might want to keep this for older versions of Clickhouse without the `is_deleted` column. @aadant  WDYT?

Reply:
Yes, actually I have a better version of this code that uses ANTLR for parsing and falls back to the regexp code.
I will send a PR soon. It should fix the comment issue !

Reply:
see https://github.com/Altinity/clickhouse-sink-connector/pull/310 for a similar fix (except for the comment)
once we reconcile this PR and #310, we can merge this one that uses a proper DDL parser and falls back to the original regexp parser.